### DISCUSSION TITLE
"Easy pick" label to increase contributions

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#50737] [Q&A]
I think/hope there are developers that would like to contribute (more) to Symfony but they abandon to pick up an issue because they can't estimate how much work/time it will require or just don't understand the issue.
What do you thing about new label for issues that would mean this issue is relatively easy to pick up? Somewhere on the GitHub I saw very nice and catchy label called `easy pick`.

I know Symfony has `Good first issue` but I'm not sure this means the same what `easy pick`, because the former may be understood as issue for the very first issue (for newcomers) and the latter means you have to know some Symfony, already contributed a few times to an open source project and the only problems are the ones I mentioned at the first paragraph.

However, if in your opinion both labels are equivalent then maybe `Good first issue` should be resurrected and used more often as I can barely see this label being added.

**Comment by stof** [comment]
The issue with this kind of label is that easy issues often don't stay there long (precisely because they are easy to do).
